conf/ds-present-environment.yaml.SAMPLE

```diff
@@ -0,0 +1,125 @@
+# * folder, serving sample files
+storages:
+  # There can be multiple defined storages. The outer YAML key is used as the ID for the storage.
+  - main:
+      # If default is true, this will be the default storage for collections.
+      # The default storage choice can be overridden in a collection with the 'storage' property
+      default: true
+      # Backends contains a list of one or more implementations and their configuration, with the primary key being
+      # the implementation (aka type) of the backend
+      #
+      # Available backends are
+      # ds-storage: A local instance of https://github.com/kb-dk/ds-storage/
+      # folder:     Local files under the stated folder. Use only for test as the implementation in unsecure
+      backends:
+        # The ds-storage backend connects to a running https://github.com/kb-dk/ds-storage/ instance
+        - ds-storage:
+            # The name of the machine running the service. Mandatory
+            dbserverurl: 'http://localhost:9072/ds-storage/v1/'
+
+# Settings for handling export of multiple records
+records:
+  # If true, export is stopped if a transformation fails for any record
+  # If false, failed records are logged and skipped
+  # Default: true
+  stoponerrors: true
+
+
+# Origins are to be set in ds-present-kb-collections-yaml if possible, as that file overwrites the origins defined here
+# The origins available for this instance of ds-present
+# General note: All origins are expected to support at least the views raw, mods, jsonld and solrjson.
+# If any of those are not supported, the transformer 'fail' should be used with a telling message.
+#origins:
+  # Straight forward origin with the core format being MODS
+  # Uses the default storage, which connects to a running ds-storage instance
+#  - remote: # ds-present internal
+#      prefix: 'doms.radio' # The prefix to match on incoming IDs. Selectet to use ds-storage without any tweaks
+#      description: 'Images from the collections at the Royal Danish Library'
+#   Different views on the material. Note that all origins has at least mods, jsonld and solrjson as views
+#      views:
+#        - raw:
+#            mime: 'text/plain'
+#            transformers:
+#              - identity:
+#        - MODS:
+#            mime: 'application/xml'
+#            transformers:
+#             - identity: # No transformation steps as mods is the core format for images-dsfl
+#        - JSON-LD:
+#            mime: 'application/json'
+#            transformers:
+#              - xslt:
+#                  stylesheet: 'xslt/mods2schemaorg.xsl'
+#                  injections:
+#                    - imageserver: 'https://example.com/imageserver'
+#        - SolrJSON:
+#            mime: 'application/json'
+#            transformers:
+#              - xslt:
+#                  stylesheet: 'xslt/mods2solr.xsl'
+#                  injections:
+#                    - imageserver: 'https://example.com/imageserver'
```

Comment:
This is a leftover from before we had `${path:...}` expansion for YAML. TEK will (probably) never need to change this. Please remove